docker/edge/http/edge_start.sh

```diff
@@ -0,0 +1,9 @@
+if [ $# -eq 0 ]; then
```

Comment:
mention in the start of the script  as a comment what should be input parameter and what this script is designed for.

Reply:
Comment Incorporated